tools/autodeployment/src/sibling-checker/index.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-expressions */
+const fetch = require('node-fetch');
+const { setIntervalAsync, clearIntervalAsync } = require('set-interval-async/dynamic');
+
+const server =
+  process.DEPLOY_TYPE === 'production'
+    ? 'https://telescope.cdot.systems/health.'
+    : 'https://dev.telescope.cdot.systems/health.';
+
+let timerOne;
+let timerTwo;
+let messageSent = false;
+
+const firstCheck = () => {
+  timerOne = setIntervalAsync(async () => {
+    const result = await fetch(server, {
+      method: 'head',
+      timeout: 1500,
+    });
+    if (result.status !== 200) {
+      clearIntervalAsync(timerOne);
+      secondCheck();
```

Comment:
instead of this, why don't have the 2 intervals running independently? Or maybe one interval calling the 2 checks sequentially.